sdk/src/test/java/com/mapbox/maps/NativeMapTest.kt

```diff
@@ -408,11 +408,11 @@ class NativeMapTest {
   }
 
   @Test
-  fun jumpTo() {
+  fun setCameraOptions() {
```

Comment:
just `setCamera` I guess.

Reply:
This is a test, we have a setFreeCameraOptions test as well, the naming of the test should be specific.